src/main/scala/shade/memcached/MemcachedImpl.scala

```diff
@@ -223,6 +224,64 @@ class MemcachedImpl(config: Configuration, ec: ExecutionContext) extends Memcach
     instance.shutdown(3, TimeUnit.SECONDS)
   }
 
+  /**
+   * Atomically increments the given key by a non-negative integer amount
+   * and returns the new value.
+   *
+   * The value is stored as the ASCII decimal representation of a 64-bit
+   * unsigned integer.
+   *
+   * If the key does not exist and a default is provided, sets the value of the
+   * key to the provided default and expiry time.
+   *
+   * If the key does not exist and no default is provided, or if the key exists
+   * with a value that does not conform to the expected representation, the
+   * operation will fail.
+   *
+   * If the operation succeeds, it returns the new value of the key.
+   *
+   * Note that the default value is always treated as None when using the text
+   * protocol.
+   *
+   * The expiry time can be Duration.Inf (infinite duration).
+   */
+  def increment(key: String, by: Long, default: Option[Long], exp: Duration): Future[Long] =
+    instance.realAsyncMutate(withPrefix(key), by, Mutator.incr, default, exp, config.operationTimeout) map {
+      case SuccessfulResult(givenKey, value) =>
```

Comment:
Not a big deal, but since `givenKey` isn't being used we can just replace its binding with `_` to make it obvious we are ignoring it.